tutor/specs/components/task-plan/homework/__snapshots__/add-exercises.spec.cjsx.snap

```diff
@@ -81,8 +81,18 @@ exports[`choose exercises component matches snapshot 1`] = `
             />
           </div>
           <span>
-            Tutor Selections
+            OpenStax Tutor Selections
           </span>
+          <button
+            aria-describedby="homework-selections-popover"
+            className="btn btn-link"
+            disabled={false}
+            id="homework-selections-trigger"
+            onClick={[Function]}
+            type="button"
+          >
+            What are these?
+          </button>
         </div>
         <div
```

Comment:
Are you working off a mockup on this?  The new "What are these?" pushes the bar down and makes the vertical separator bars no longer fill the height.   Maybe we could make the popover trigger be the (?) icon and have it appear after the "OpenStax Tutor Selections" so we don't need a new line. 
![screen shot 2017-06-16 at 9 25 05 am](https://user-images.githubusercontent.com/79566/27230734-dc222058-5275-11e7-9df2-7e60b2a751f4.png)